discordclient/commands/commands.go

```diff
@@ -20,12 +21,38 @@ var commandHandlers = map[string]func(s *dgo.Session, i *dgo.InteractionCreate){
 	repositoryLinkCommand.Name: repositoryLinkHandler,
 	alumniLinkCommand.Name:     alumniLinkHandler,
 	socialNetworkCommand.Name:  socialNetworkHandler,
+	createEventsCommand.Name:   createEventsHandler,
 }
 
 // CommandsHandlers is the handler of slash commands
 func CommandsHandlers(s *dgo.Session, i *dgo.InteractionCreate) {
-	if h, ok := commandHandlers[i.ApplicationCommandData().Name]; ok {
-		h(s, i)
+	switch i.Type {
```

Comment:
It will be a better idea to handle Interaction modal in an other package.
May be a backage named `modal` ?

Reply:
Yes it can be better, but i'm not really sure how to do it, the command seems to all be handled here

Reply:
For now the commands are the only handles.
What i'm talking about is completly new. It something we'll also need when we'll pass the creation of e message role as a modal instead of using arguments.